projects/core/src/routing/external-routes/external-routes.service.ts

```diff
@@ -55,9 +55,7 @@ export class ExternalRoutesService {
    * Returns the URL matcher for the external route
    */
   protected getUrlMatcher(): UrlMatcher {
-    const matcher = this.matcherFactory.getGlobUrlMatcher(
-      this.internalUrlPatterns
-    );
-    return this.matcherFactory.getOppositeUrlMatcher(matcher); // the external route should be activated only when it's NOT an internal route
+    const matcher = this.urlMatcherService.fromGlob(this.internalUrlPatterns);
```

Comment:
i have a hard time to understand the meaning of `fromGlob`

Reply:
Thanks for feedback. Maybe I went too far with shortening the method names.
`UrlMatcherService.getFromGlob()` would help?
Similar for:
`UrlMatcherService.getFromPaths()`
`UrlMatcherService.getCombined()`